### PR TITLE
SASS Stylesheet Update

### PR DESCRIPTION
**Pull request by leostudiooo**
1. Updated `font-family` to the sans-serif fonts: 
`-apple-system, BlinkMacSystemFont, 'Noto Sans', 'Source Han Sans', 'Microsoft YaHei UI', 'Microsoft YaHei', 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif`
2. Created settable base `z-index` named `$least-zindex` in `_variables.sass`.

### CODE REVIEW

File: src/sass/player.sass [hunk 1]
```diff
@@ -7,13 +7,13 @@
         transform: rotate(360deg)
 
 .penguin-player
-    font-family: "Microsoft YaHei UI", "Microsoft YaHei", "黑体"
+    font-family: -apple-system, BlinkMacSystemFont, 'Noto Sans', 'Source Han Sans', 'Microsoft YaHei UI', 'Microsoft YaHei', 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
```
Comment (LuoTR) [inline src/sass/player.sass:10]: Have you tested the player with all of these fonts?

Comment (leostudiooo) [reply src/sass/player.sass:10]: yup but -apple-system and BlinkMac

Comment (leostudiooo) [reply src/sass/player.sass:10]: This is what vscode tell me to do

Comment (leostudiooo) [reply src/sass/player.sass:10]: but it should be no problem when running on macOS.

Comment (LuoTR) [reply src/sass/player.sass:10]: Well, then I will test it myself later

**Pull request merged**